src/components/Faq/FaqNav.js

```diff
@@ -5,13 +5,13 @@ import "./FaqNav.css";
 import Button from "../Button";
 
 const FaqNav = ({ navData, pathname, lang }) => {
-  let pathRoot = `/refugee-guide`;
+  let pathRoot = `/ua/refugee-guide`;
   switch (lang) {
```

Comment:
maybe it's possible to do this without switch case ?

Reply:
Once we decide on the correct way to approach this, I expect there to be a global constant/handler for this.

Right now changed the current ukrainian sites to have a `/ua/` prefix in pathnames purely for testing purposes.